### PR TITLE
Firm sorting scopes

### PR DESCRIPTION
**Pull request by munckymagik**
I will probably squash the last rename commit once we're done here.

### CODE REVIEW

File: app/models/firm.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ class Firm < ActiveRecord::Base
   ]
 
   scope :registered, -> { where.not(email_address: nil) }
+  scope :sorted_by_registered_name, -> { order(:registered_name) }
+
+  def self.most_recently_updated(limit: nil)
+    order(updated_at: :desc).limit(limit)
+  end
```
Comment (munckymagik) [inline app/models/firm.rb]: Made the 2nd one a class method to keep Rubocop happy: it was too long as a proc and [this](https://github.com/bbatsov/ruby-style-guide/issues/270) ...

Comment (neoeno) [inline app/models/firm.rb]: There any sense in giving this a default value? (I don't know)

**Comment by neoeno** [comment]
:+1:

**Comment by rnalexander** [comment]
:+1:

File: app/models/firm.rb (hunk 1 above)
Comment (munckymagik) [inline app/models/firm.rb]: Me neither. Currently I'm seeing the limit as an application side decision, let's see how it goes.

**Pull request merged**